platform/ios/app/MBXViewController.m

```diff
@@ -40,6 +40,7 @@ + (void)initialize
             @"MBXUserTrackingMode": @(MGLUserTrackingModeNone),
             @"MBXShowsUserLocation": @NO,
             @"MBXDebug": @NO,
+            @"MGLMapboxMetricsTestServerURL": @"https://cloudfront-staging.tilestream.net",
```

Comment:
I thought about that, but we have a variety of other legacy keys that are user-facing and we don’t want to change. Seems like it may be confusing to not be consistent.